### PR TITLE
Deprecate make/nmake/sln  C++/SYCL examples

### PR DESCRIPTION
**Pull request by maria-Petrova**
The PR aims to exclude the following daal/onedal examples from OS and binary distributions:
-  make (all OSes)
- MSVS solutions on Windows

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -1049,10 +1048,10 @@ $(RELEASEDIR.daal)/$1/$2: \
 	python ./deploy/local/generate_win_solution.py $1 $(RELEASEDIR.daal)/$1 $2 --template_name $3
```
Comment (napetrov) [inline makefile:1048]: i think that entire release VS solution should be removed(1035 - 1055) as well as ./deploy/local/generate_win_solution.py

Comment (maria-Petrova) [reply makefile]: Thanks Nikolay. Fixed.

**Comment by napetrov** [comment]
./deploy/local/generate_win_solution.py  should be removed as well

**Comment by maria-Petrova** [comment]
> ./deploy/local/generate_win_solution.py should be removed as well

do we want to remove makefile generator as sell?

**Comment by napetrov** [comment]
> > ./deploy/local/generate_win_solution.py should be removed as well
> 
> do we want to remove makefile generator as sell?

yes, let's also clean it

**Comment by maria-Petrova** [comment]
Agreed with @homksei that he will address the issues in testing.

**Review by napetrov — APPROVED** [review]

**Pull request merged**